### PR TITLE
Definitions and examples for Course related terms

### PR DESCRIPTION
**Pull request by philbarker**
small tweaks suggested after discussion about inclusion of stable Course-related terms into schema.org core. Progress on issue #195

### CODE REVIEW

File: data/ext/pending/issue-195.rdfa [hunk 1]
```diff
@@ -32,7 +32,7 @@ These are now (experimentally) in boilerplate-195.rdfa instead.
     <div typeof="rdfs:Class" resource="http://schema.org/Course">
 
         <span class="h" property="rdfs:label">Course</span>
-        <span property="rdfs:comment">A description of an educational course which may be offered as distinct instances at different times and places, or through different media or modes of study. An educational course is a sequence of one or more educational events and/or creative works which aims to build knowledge, competence or ability of learners.</span>
+        <span property="rdfs:comment">A description of an educational course which may be offered as distinct instances at which take place at different times or takeplace at different locations, or be offered through different media or modes of study. An educational course is a sequence of one or more educational events and/or creative works which aims to build knowledge, competence or ability of learners.</span>
```
Comment (danbri) [inline data/ext/pending/issue-195.rdfa]: "or takeplace at" - can you add a space? 

i.e. "or  take place at"

File: data/ext/pending/issue-195.rdfa [hunk 2]
```diff
@@ -74,15 +74,15 @@ These are now (experimentally) in boilerplate-195.rdfa instead.
     <div typeof="rdfs:Class" resource="http://schema.org/CourseInstance">
 
         <span class="h" property="rdfs:label">CourseInstance</span>
-        <span property="rdfs:comment">An instance of a [[Course]] offered at a specific time and place or through specific media or mode of study or to a specific section of students.</span>
+        <span property="rdfs:comment">An instance of a [[Course]] which is distinct from other instances because it is offered at a different time or location or through differnt media or modes of study or to a specific section of students.</span>
```
Comment (danbri) [inline data/ext/pending/issue-195.rdfa]: different

**Comment by philbarker** [comment]
fixing typos

**Comment by philbarker** [comment]
@danbri gah! 
You might want to fix app.yaml from
#application: schemaorgae
#application: webschemas
application: schema-course-extend

My fault, must have fixed it wrong.
Have now got my local repo to ignore changes in that file.

**Comment by danbri** [comment]
No worries, I'm always flipflopping that file, I should rig it as ignored too :)

**Pull request merged**